Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.9
+RUN go get -u github.com/golang/dep/cmd/dep
+WORKDIR $GOPATH/src/github.com/grafana/grafana
+COPY Gopkg.toml Gopkg.lock ./
+RUN dep ensure --vendor-only
```

Comment:
Any reason for using dep ensure instead of just copying the vendor folder?

Reply:
It ensures everything is properly locked in and it works when adding dependencies. Also allows removal of the vendor folder, but that is more of a personal preference (this will work just fine with the vendor folder).

Reply:
I prefer to just copy the vendor folder. Should be much faster and at some point I would like to use this for CI/CD :)

Reply:
You could include vendor in the copy line, add a USE_VENDOR ARG, then make running dep conditional on that arg being in the environment passed by RUN

Reply:
Sounds good